src/main/java/entities/Developer.java

```diff
@@ -0,0 +1,30 @@
+package entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+
+import javax.persistence.*;
+
+@Log4j
+@Getter
+@Setter
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "developers")
+public class Developer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+    @Column(name = "Name")
+    private String name;
+    @Column(name = "Name")
+    private String city;
+    @Column(name = "Age")
+    private int age;
+    @Column(name = "salary")
+    private int salary;
```

Comment:
In your project is it possible situation when in DB salary is absent? 
If it possible use wrapper to primitive.
https://stackoverflow.com/questions/7506802/using-wrapper-integer-class-or-int-primitive-in-hibernate-mapping
The same to other entity classes